test/data_files/sample_schemas/project.json

```diff
@@ -0,0 +1,592 @@
+{
+    "title": "Project",
+    "$id": "/profiles/project.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "required": [
+        "name",
+        "title"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "name",
+        "title",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/modified"
+        },
+        {
+            "$ref": "mixins.json#/tags"
+        },
+        {
+            "$ref": "mixins.json#/status"
+        },
+        {
+            "$ref": "mixins.json#/static_embeds"
+        }
+    ],
+    "type": "object",
+    "properties": {
+        "static_headers": {
```

Comment:
Same as with User, a lot of this schema stuff can probably be eliminated and just test representative cases.